src/maps/review-and-submit-test-summary.map.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@nestjs/common';
+
+import { BaseMap } from '@us-epa-camd/easey-common/maps';
+import { ReviewAndSubmitTestSummaryDTO } from '../dto/review-and-submit-test-summary.dto';
+import { TestSummaryReviewAndSubmit } from '../entities/test-summary-review-and-submit.entity';
+
+@Injectable()
+export class ReviewAndSubmitTestSummaryMap extends BaseMap<
+  TestSummaryReviewAndSubmit,
+  ReviewAndSubmitTestSummaryDTO
+> {
+  public async one(
+    entity: TestSummaryReviewAndSubmit,
+  ): Promise<ReviewAndSubmitTestSummaryDTO> {
+    return {
```

Comment:
let's stay consistent with the code properties and spell out the word "Code"
orisode
testTypeCode
evalStatusCode
submissionAvailabilityCode